src/modules/searchform.js

```diff
@@ -0,0 +1,14 @@
+import displayMeals from './displayMeals';
+
+const URL = 'https://www.themealdb.com/api/json/v1/1/search.php?s=';
+
+const form = document.querySelector('.search-form');
+const mealInput = document.querySelector('.meal-input');
+
+form.addEventListener('keyup', (e) => {
+  e.preventDefault();
+  const { value } = mealInput;
+  console.log(value);
```

Comment:
### OPTIONAL

_**I write it as an optional but this is a requirement that is not mentioned in the last review. So please fix it.**_

- [ ] You should delete the console.log as last TSE said. Last TSE said optional but this is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=Do%20not%20commit%20console.log%20to%20your%20repo.%20It%E2%80%99s%20ugly%2C%20it%20kills%20performance%20and%20it%20can%20make%20confidential%20data%20be%20visible%20to%20anyone%20using%20the%20browser%20tools%20to%20look%20at%20your%20website.).